src/NuGet.Core/NuGet.LibraryModel/CentralPackageVersion.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using NuGet.Shared;
+using NuGet.Versioning;
+
+namespace NuGet.LibraryModel
+{
+    public class CentralPackageVersion : IEquatable<CentralPackageVersion>
```

Comment:
Is CentralPackageVersion really a core library model concept?  Or is it an implementation detail of restore, so it could be an internal class in NuGet.Commands?

Reply:
It's used in the package spec to persist what the central versions were at time of restore. So, yes, it's core to the library model.